src/menus/icon-menu.jsx

```diff
@@ -24,14 +24,16 @@ let IconMenu = React.createClass({
     ]),
     onItemKeyboardActivate: React.PropTypes.func,
     onItemTouchTap: React.PropTypes.func,
-    menuStyle: React.PropTypes.object
+    menuStyle: React.PropTypes.object,
+    timeout: React.PropTypes.number
```

Comment:
I would agree, makes it more clear what the property is used for. @cgestes can you add this new prop to the docs?